### PR TITLE
kiss: Cleanup

### PR DESCRIPTION
**Pull request by clifordsab**

### CODE REVIEW

File: kiss [hunk 1]
```diff
@@ -782,7 +781,7 @@ pkg_swap() {
     [ -f "$alt" ] || [ -h "$alt" ] ||
         die "Alternative '$1 $2' doesn't exist"
 
-    if [ -f "$2" ]; then
+    [ -f "$2" ] && {
```
Comment (dylanaraps) [inline kiss:784]: This should stay as-is due to `set -e`.

File: kiss [hunk 2]
```diff
@@ -395,6 +394,7 @@ pkg_fixdeps() {
     if [ -f depends ]; then
         cp -f depends "$mak_dir/d"
         dep_file=$mak_dir/d
+
```
Comment (dylanaraps) [inline kiss:397]: This was fine before. :+1:

File: kiss [hunk 3]
```diff
@@ -915,10 +915,10 @@ pkg_remove() {
     # an incomplete package installed.
     trap '' INT
 
-    if [ -x "$sys_db/$1/pre-remove" ]; then
+    [ -x "$sys_db/$1/pre-remove" ] && {
         log "$1" "Running pre-remove script"
         "$sys_db/$1/pre-remove" ||:
-    fi
+    }
```
Comment (dylanaraps) [inline kiss:921]: Was fine before (due to `set -e`).

File: kiss [hunk 4]
```diff
@@ -1087,10 +1087,10 @@ pkg_install() {
     # we no longer need to block 'Ctrl+C'.
     trap pkg_clean EXIT INT
 
-    if [ -x "$sys_db/$pkg_name/post-install" ]; then
+    [ -x "$sys_db/$pkg_name/post-install" ] && {
         log "$pkg_name" "Running post-install script"
         "$sys_db/$pkg_name/post-install" ||:
-    fi
+    }
```
Comment (dylanaraps) [inline kiss:1093]: Was fine before (`set -e`).

**Comment by clifordsab** [comment]
Force-pushed corrections. :thumbsup:

**Comment by dylanaraps** [comment]
Thanks

**Pull request merged**